Regression/WarpX-tests.ini

```
@@ -1937,6 +1937,22 @@ analysisRoutine = Examples/Tests/collision/analysis_collision_rz.py
 aux1File = Regression/PostProcessingUtils/post_processing_utils.py
 tolerance = 1.e-15
 
+[Proton_Boron_Fusion_3D]
+buildDir = .
+inputFile = Examples/Modules/nuclear_fusion/inputs_3d
+runtime_params =
```

Because we use `boron1.injection_style = "NRandomPerCell"` (ics=1) and use more than one box with particles (dyn sched.):
```suggestion
runtime_params = warpx.do_dynamic_scheduling=0 warpx.serialize_initial_conditions=1
```